haystack/document_stores/dc.py

```diff
@@ -0,0 +1,163 @@
+from typing import List, Optional, Union, Dict, Any, Generator
+
+import json
+import logging
+import requests
+
+from haystack.document_stores import BaseDocumentStore
+from haystack.schema import Document, Label, MultiLabel
+
+DEFAULT_API_ENDPOINT = f"DC_API/v1"
```

Comment:
Could we maybe use `DEFAULT_API_ENDPOINT = os.getenv("DEFAULT_API_ENDPOINT",f"DC_API/v1")` and overwrite the configuration via environment variables ?  This would also be very helpful for the `api_key`, because we also use the these key in other places and don't always want to add it manually by calling `os.getenv...`.

Reply:
Done: api_key is read from `DEEPSET_CLOUD_API_KEY` and api_endpoint is read from `DEEPSET_CLOUD_API_ENDPOINT` env variables.